### PR TITLE
add a note explaining the sed command

### PR DESCRIPTION
**Pull request by jdkandersson**
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

During user testing I got feedback that it would be nice to explain the `sed` command updating the `base` for the chisel section of the tutorial explaining what the command does.

### CODE REVIEW

File: docs/tutorials/getting-started-with-flask.rst [hunk 1]
```diff
@@ -12,6 +12,13 @@ Setup
 
 .. include:: /reuse/tutorials/setup.rst
 
+.. note::
+    This tutorial required version ``1.5.2`` or later for Rockcraft. Check the
```
Comment (tigarmo) [inline docs/tutorials/getting-started-with-flask.rst:16]:
```suggestion
    This tutorial requires version ``1.5.2`` or later for Rockcraft. Check the
```

File: docs/tutorials/getting-started-with-flask.rst [hunk 2]
```diff
@@ -12,6 +12,13 @@ Setup
...
+    version of Rockcraft using ``rockcraft --version``. If you have an older
+    version of Rockcraft installed, use
+    ``sudo snap refresh rockcraft --channel latest/stable`` to get the latest
+    stable version of Rockcraft.
```
Comment (tigarmo) [inline docs/tutorials/getting-started-with-flask.rst:20]:
```suggestion
    stable version.
```

I think this says "version of Rockcraft" enough times to make it clear

**Review by tigarmo — CHANGES_REQUESTED** [review]

**Review by lengau — APPROVED** [review]

File: docs/tutorials/getting-started-with-flask.rst [hunk 3]
```diff
@@ -12,6 +12,13 @@ Setup
 
 .. include:: /reuse/tutorials/setup.rst
 
+.. note::
+    This tutorial requires version ``1.5.2`` or later for Rockcraft. Check the
+    version of Rockcraft using ``rockcraft --version``. If you have an older
```
Comment (SamirPS) [inline docs/tutorials/getting-started-with-flask.rst:17]: 'check your version with ... ' sound better no?

**Review by lengau — APPROVED** [review]

File: docs/tutorials/getting-started-with-flask.rst (hunk 3 above)
Comment (jdkandersson) [reply docs/tutorials/getting-started-with-flask.rst:22]: Updated

**Review by SamirPS — APPROVED** [review]

File: docs/tutorials/getting-started-with-flask.rst [hunk 4]
```diff
@@ -249,6 +256,11 @@ In ``rockcraft.yaml``, change the ``base`` to ``bare`` and add
     :end-before: [docs:change-base-end]
     :dedent: 2
 
+.. note::
+    The ``sed`` command replaces the current ``base`` in ``rockcraft.yaml`` with
+    the ``bare`` base. The command also adds a ``build-base`` which is required
+    for the ``bare`` base.
```
Comment (cjdcordeiro) [inline docs/tutorials/getting-started-with-flask.rst:262]:
```suggestion
    when using the ``bare`` base.
```

**Review by cjdcordeiro — APPROVED** [review]

File: docs/tutorials/getting-started-with-flask.rst (hunk 4 above)
Comment (jdkandersson) [reply docs/tutorials/getting-started-with-flask.rst]: Updated

**Comment by jdkandersson** [comment]
@tigarmo it seems that the failed spread tests are unrelated?

**Pull request merged**